EpiAware/docs/src/examples/getting_started.jl

```diff
@@ -236,42 +240,61 @@ However, we now treat the generated data as `truth_data` and make inference with
 We do the inference by MCMC/NUTS using the `Turing` NUTS sampler with default warm-up steps.
 "
 
-# ╔═╡ c8ce0d46-a160-4c40-a055-69b3d10d1770
-truth_data = generated_obs
+# ╔═╡ 7e48a4c5-cd30-4377-8a98-e0c23f2dc31e
+truth_data = random_epidemic.y_t
+
+# ╔═╡ 272e6798-1151-486f-9667-924dbc63bd69
+inference_mdl = fix(
+    make_epi_aware(truth_data, time_horizon;
+        epi_model = epi_model,
+        latent_model = rwp,
+        observation_model = obs_model),
+    (init_rw = 0.0,)
+)
 
-# ╔═╡ 4a4c6e91-8d8f-4bbf-bb7e-a36dc281e312
+# ╔═╡ 4298f0ec-f6df-42ee-aa28-f7ed60f1e530
 md"
-The observation model supports partially complete data. To test this we set some of the generated observations to be `missing`.
+### Initialising inference
+
+It is possible for the default warm-up process for NUTS to get stuck in low probability or otherwise degenerate regions of parameter space.
+
+To make NUTS more robust we provide `manypathfinder`, which is built on pathfinder variational inference through the [Pathfinder.jl](https://mlcolab.github.io/Pathfinder.jl/stable/). `manypathfinder` runs `nruns` pathfinder processes on the inference problem and returns the pathfinder run with maximum estimated ELBO.
+
+`manypathfinder` differs from `Pathfinder.multipathfinder`; `multipathfinder` is aimed at sampling from a potentially non-Gaussian target distribution which is first approximated as a uniformly weighted collection of normal approximations from pathfinder runs. `manypathfinder` is aimed at moving rapidly to a 'good' part of parameter space, and is robust to runs that fail.
 "
 
-# ╔═╡ 259a7042-e74f-43c7-aeb4-97a3beeb7776
-let
-    truth_data = Union{Int, Missing}[truth_data...]
-    truth_data[vcat([3, 5], 10:20)] .= missing
-end
+# ╔═╡ 40ebd47a-4a08-4a46-a727-26347d3fca51
+best_pf = manypathfinder(inference_mdl; nruns = 20);
 
-# ╔═╡ 32638954-2c99-4d4e-8e03-52154030c657
+# ╔═╡ b7d9a56a-b2d5-4595-a6b9-9cd5fa6b1445
 md"
-We now make the model but fixing the initial condition of the random walk to be 0.
+We can use draws from the best pathfinder run to initialise NUTS.
 "
 
-# ╔═╡ b4033728-b321-4100-8194-1fd9fe2d268d
+# ╔═╡ cdd805e2-b00c-4522-9261-1819c6a195eb
+best_pf.draws_transformed
 
-inference_mdl = fix(
-    make_epi_aware(truth_data, time_horizon; epi_model = epi_model,
-        latent_model = rwp, observation_model = obs_model),
-    (rw_init = 0.0,)
-)
+# ╔═╡ e847b0b6-9d70-46ba-bec6-1e3fa676a33c
+init_params = collect.(eachrow(best_pf.draws_transformed.value[1:4, :, 1]))
```

Comment:
Can we have a helper function for this or make it part of the output of best_pf?  I don't love reading this or the thought of having to repeatedly write it down.

Reply:
Thats an issue now